common/Reducers.jsm

```diff
@@ -172,6 +180,8 @@ function TopSites(prevState = INITIAL_STATE.TopSites, action) {
       }
       newRows = prevState.rows.filter(site => action.data.url !== site.url);
       return Object.assign({}, prevState, {rows: newRows});
+    case at.UPDATE_SEARCH_SHORTCUTS:
+      return Object.assign({}, prevState, action.data);
```

Comment:
I would do `return {...prevState, searchShortcuts: action.data.searchShortcuts}` here just so this is easier to read / figure out what's going on